docker/Dockerfile.arm64

```diff
@@ -1,5 +1,8 @@
+# Note: Due to https://github.com/gotify/server/issues/350 this Dockerfile
```

Comment:
Well, it can be added, but then the ci build must be adjusted accordingly. I don't think such a comment is needed, because the issue already exists.